libnymea/interfaces/colortemperaturelight.json

```diff
@@ -5,7 +5,8 @@
             "name": "colorTemperature",
             "type": "int",
             "minimumValue": "any",
-            "maximumValue": "any"
+            "maximumValue": "any",
```

Comment:
having here a min max value is for having a slider in the client application (I assume), what does the client see in this case? min max are actually numbers, not strings, so this could cause problems regarding consistency?

Reply:
Not sure if I understand the question properly. Yes, min/max are numbers, so is the type of this state. A lamp might support color temperature from 4000 to 5000 kelvin (well, so it sets that here, another lamp might do 3500 - 4900).

Why should it be strings?

Reply:
I understand the use of `"any"` in the interface description. I think it should be described that the value has to be an integer and can be any value. Meaning: no restrictions regarding the actual value of the min/max. At least for me this was not clear and new.